### DISCUSSION TITLE
The extension is not working

### DISCUSSION THREAD

**Discussion by sahaib** [Authenticator-Extension/Authenticator#1207] [General]
<img width="1680" alt="Screenshot 2024-06-16 at 4 13 56 PM" src="https://github.com/Authenticator-Extension/Authenticator/assets/11386285/55dae4d3-4356-4b01-8507-ba1fba6bc10d">

**Comment by mymindstorm** [comment]
Please follow [these steps](https://authenticator.cc/docs/en/debug-info).
